### ISSUE TITLE
ng build --prod error

### ISSUE THREAD

**Issue by myelxx** [neuronetio/angular-gantt-schedule-timeline-calendar#16]
![image](https://user-images.githubusercontent.com/35130442/79832418-f897dd00-83db-11ea-8647-ab6c0fe32630.png)

says: GSTCComponent is a part of two 2 modules

**Comment by JoniVR** [comment]
Same issue here, I'd love to use it but right now it seems pretty broken.. I've been trying to upgrade to angular 9 but I keep hitting walls.

**Comment by JoniVR** [comment]
I think I managed to work around this issue by doing this:
```ts
import { GSTCComponent, AngularGanttScheduleTimelineCalendarModule } from 'angular-gantt-schedule-timeline-calendar';
```

and then instead of adding it to the app.module declarations I added it to the imports:

```ts  
imports: [
    AngularGanttScheduleTimelineCalendarModule
  ],
```

**Comment by myelxx** [comment]
> Same issue here, I'd love to use it but right now it seems pretty broken.. I've been trying to upgrade to angular 9 but I keep hitting walls.

thank you for your suggestion. it works for me. i've been struggling to solve and your solution perfectly works. thanks!

**Comment by charlypeluch** [comment]
> Same issue here, I'd love to use it but right now it seems pretty broken.. I've been trying to upgrade to angular 9 but I keep hitting walls.

Did you finally manage to run the gantt library in Angular 9?

**Comment by JoniVR** [comment]
> Did you finally manage to run the gantt library in Angular 9?

Yes, it just worked for me using the solution I provided above without issues in Angular 9. 

Honestly you could also just create a new component and copy-paste the contents of [this file](https://github.com/neuronetio/angular-gantt-schedule-timeline-calendar/blob/master/projects/angular-gantt-schedule-timeline-calendar/src/lib/angular-gantt-schedule-timeline-calendar.component.ts) inside of it and change the GSTC import to import from your node modules (after you install the [javascript version](https://github.com/neuronetio/gantt-schedule-timeline-calendar))

That way you won't have to deal with this project not being updated to Angular 9 and you can manage the wrapper yourself. That's all this repo essentially does, wrap the project for Angular.

**Comment by alexalloza** [comment]
Hi, i dont understand how do you ran the gant library in Angular 9. In my console throws me a AOT error.

How do you import the library?

Thanks in advance @JoniVR

**Comment by JoniVR** [comment]
What exact error are you getting? I would just recommend wrapping it yourself like I explained above, seems easier and better to maintain long term.

**Comment by oguzhan80** [comment]
@JoniVR thanks for the wrapper hint, works like a charm!
